Makefile

```diff
@@ -44,7 +44,7 @@ $(binary_dirs): noop
 	$(docker) /bin/sh -c "cd $@; go build -i"
 
 test:
-	$(docker) go test $$(glide novendor)
+	$(docker) go test -race $$(glide novendor)
```

Comment:
I recall removing this previously for performance reasons - we will run it again every time?

https://golang.org/doc/articles/race_detector.html#Runtime_Overheads

Reply:
it doesn't appear to be as terrible as i suspected: 0m30.679s for both race and coverage now